presentation/components/src/presentation-components/tree/controlled/PresentationTreeNodeRenderer.scss

```diff
@@ -0,0 +1,9 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import "~@itwin/itwinui-css/scss/style/theme";
+
+.presentation-custom-node{
+  color: $iui-text-color-muted--light;
```

Comment:
I think this should be themed so it should look like this:
`@include themed {
    color: t(iui-text-color-muted);
  }`

More info: https://github.com/iTwin/iTwinUI#using-sass